docs/accessing-the-cluster.md

```diff
@@ -1,26 +1,20 @@
 # Accessing the Cluster
 
-## Using the Kubernetes proxy to access the cluster
+ * [Using the Kubernetes proxy](#proxy)
+ * [Requesting redirects](#redirect)
+
+## <a name="proxy"></a>Using the Kubernetes proxy to access the cluster
 Information about the cluster can be accessed by using a proxy URL and the cluster authentication keys.
-For example, for a cluster that has cluster-level logging enabled (using Elasticsearch), you can retrieve information about the Elasticsearch logging on that cluster.
+For example, if a cluster has cluster-level logging enabled (using Elasticsearch), you can retrieve information about the Elasticsearch logging on that cluster through a proxy URL. 
+
+### Retrieving the authentcation keys and proxy URLs
+Use `kubectl` commands to retrieve the access information.
 
-To retrieve the configuration details about your cluster, including the address of the Kubernetes master cluster and the cluster's authentication keys (username, password, and access token), run the following command:
+To retrieve the authentication keys for your clusters, run the following command:
 ```
 $ kubectl config view
 {
```

Comment:
I think it might be best if you ran the commands for yourself and then copy and pasted and edited (to remove passwords, token, certs) yourself to keep things as faithful as possible.